installation/Developer-guide.md

```diff
@@ -66,6 +66,19 @@ Navigate to your specific directory using `cd` command and run
 mkdir directory-name
 ```
 
+### Running minion jobs
+
```

Comment:
Can you add a sentence to explain what those minion jobs are? e.g. something like "Import and export tasks that can take a long time to run are queued and processed using the Minion job queue."

Reply:
Done